transmutagen/gensolve.py

```diff
@@ -5,6 +5,15 @@
 
 from jinja2 import Environment
 from sympy import im
+import numpy as np
+
+try:
+    import pyne.utils
+    import pyne.data
+    from pyne import nucname
+    HAVE_PYNE = True
+except ImportError:
+    HAVE_PYNE = False
```

Comment:
Yeah, you are right. I didn't know if it was a hard dep when I first started this branch.